### PR TITLE
Pset_AlignmentVerticalSegmentCommon and Pset_AlignmentCantSegmentCommon

### PR DESCRIPTION
**Pull request by czapplitec**
Propose Pset_AlignmentVerticalSegmentCommon and Pset_AlignmentCantSegmentCommon

### CODE REVIEW

File: IFC4x3/Properties/e/EndElevation_3OTOsjYu9DUAR5D1hN5Xdg/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="EndElevation_3OTOsjYu9DUAR5D1hN5Xdg" Name="EndElevation" UniqueId="d8758dad-8b82-4d78-a6c5-341ad71619ea" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/e/EndElevation_3OTOsjYu9DUAR5D1hN5Xdg/DocProperty.xml:2]: Somehow the PrimaryDataType is missing.

File: IFC4x3/Properties/c/CantEquilibrium_30SDceNBr6DB9tffNjbauL/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CantEquilibrium_30SDceNBr6DB9tffNjbauL" Name="CantEquilibrium" UniqueId="c070d9a8-5cbd-4634-b277-a695ed964e15" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/c/CantEquilibrium_30SDceNBr6DB9tffNjbauL/DocProperty.xml:2]: PrimaryDataType seems missing.

File: IFC4x3/Properties/e/EndSmoothingLength_2EtJ89wAfDgPQfnCNoxkN6/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Length for the circular transition at the end of the cant segment, measured from the start of the circular transition to the end of the segment.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/e/EndSmoothingLength_2EtJ89wAfDgPQfnCNoxkN6/Documentation.md:1]: Is it really a circular transition or is it a curvature smoothing corresponding to a "circular transition change of curvature"?

**Review by SergejMuhic — CHANGES_REQUESTED** [review]
Looks good. One remark regarding transition and seems two PrimaryDataTypes are missing.

File: IFC4x3/Properties/e/EndSmoothingLength_2EtJ89wAfDgPQfnCNoxkN6/Documentation.md [hunk 4]
```diff
@@ -1 +1 @@
-Length for the circular transition at the end of the cant segment, measured from the start of the circular transition to the end of the segment.
+Length for the circular transition change of curvature placed at the end of the cant segment, measured from the start of the circular transition change of curvature to the end of the cant segment.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/e/EndSmoothingLength_2EtJ89wAfDgPQfnCNoxkN6/Documentation.md:1]: Not sure I understand this fix. What do you think of the proposed amend?
```suggestion
Length for the circular transition change of curvature at the end of the cant segment, measured from the start of the circular transition change of curvature to the end of the cant segment.
```

File: IFC4x3/Properties/s/StartSmoothingLength_01PipsVZr9Q8Qgw9_KMfFV/Documentation.md [hunk 5]
```diff
@@ -1 +1 @@
-Length for the circular transition at the start of the cant segment, measured from the start of the cant segment to the end of the circular transition.
+Length for the circular transition change of curvature placed at the start of the cant segment, measured from the start of the cant segment to the end of the circular transition change of curvature.
```
Comment (SergejMuhic) [inline IFC4x3/Properties/s/StartSmoothingLength_01PipsVZr9Q8Qgw9_KMfFV/Documentation.md:1]: Same suggestion as above.
```suggestion
Length for the circular transition change of curvature at the start of the cant segment, measured from the start of the cant segment to the end of the circular transition change of curvature.
```

**Review by SergejMuhic — COMMENTED** [review]
Proposal.

File: IFC4x3/Properties/e/EndSmoothingLength_2EtJ89wAfDgPQfnCNoxkN6/Documentation.md (hunk 4 above)
Comment (czapplitec) [reply IFC4x3/Properties/e/EndSmoothingLength_2EtJ89wAfDgPQfnCNoxkN6/Documentation.md:1]: OK for me, thanks!

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**